tests/integrations/celery/test_update_celery_task_headers.py

```diff
@@ -77,33 +77,6 @@ def test_span_with_transaction(sentry_init):
             )
 
 
-def test_span_with_no_transaction(sentry_init):
```

Comment:
I deleted several tests in this file because they no longer seem to make much sense with the new behavior. Do you agree, or should we rewrite these tests somehow? It looks like we are already doing similar tests with the span enclosed in a transaction

Reply:
As the coverage of the changed code is still 100% I think it is ok.